chirp/drivers/kenwood_live.py

```diff
@@ -395,8 +395,12 @@ def set_memory(self, memory):
 class THD7Radio(KenwoodOldLiveRadio):
     """Kenwood TH-D7"""
     MODEL = "TH-D7"
+    HARDWARE_FLOW = False
+    WANTS_DTR = False
+    WANTS_RTS = False
```

Comment:
Do you actually need all three of these or just `HARDWARE_FLOW`? The `WANTS_` flags are kinda weird and not actually related to hardware flow signaling. Without them in some cases some USB chips won't do *anything*. Any chance I could get you to try this with a linux machine to confirm?

Reply:
Testing without the WANTS_* now ... IIRC, I threw all three in together because it was waiting for CD to be asserted.

Reply:
Okay, that *should* only be tied to `HARDWARE_FLOW`. The `WANTS` ones control whether or not DTR and RTS are asserted when the port is open, which some chips require to transmit anything, even if hardware flow control is not in use. FreeBSD could definitely be different, but hopefully not.

Reply:
Yeah, it appears to work fine with just `HARDWARE_FLOW = False`... pushed.